randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -222,23 +225,23 @@ Extra - asset_id: 4
 > Door to Attic; Heals? False
   * Layers: default
   * Normal Door to Attic/Door to West Ocean
-  > Door to Bowling Alley Cave
-      All of the following:
-          Morph Ball
-          Morph Ball Bombs or Power Bombs or Spring Ball
   > Door to Bowling Alley
       Trivial
   > Pickup (Topmost Missile Expansion)
       Any of the following:
-          # Knowledge (Intermediate): Trippers can be brought over to right platforms by keeping them on screen. (jump and aim up)
-          Hi-Jump Boots or Space Jump or Knowledge (Intermediate)
+          Hi-Jump Boots or Space Jump
+          # Shinespark
           Speed Booster and Shinespark (Beginner) and Normal Damage ≥ 36
           # Wall jump off of Trippers
           Ice Beam and Wall Jumps (Intermediate)
           # IBJ
           Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
+          # Knowledge (Intermediate): Trippers can be brought over to right platforms by keeping them on screen. (jump and aim up)
+          Knowledge (Intermediate)
+          # Speedboost and jump. TODO: better video - https://youtu.be/t5Va2J2zmiQ
+          Speed Booster and Movement (Beginner)
   > Pickup (Morph Ball Maze Missile Expansion)
-      Morph Ball and Super Missile
+      Super Missile and Can Lay Any Bombs
 
```

Comment:
Don't need bombs here. Just a super for the block and morph. The same with leaving, but leaving would require a way to get back into the tunnel.

Reply:
![image](https://github.com/randovania/randovania/assets/38186597/b5b9b313-bbaf-4a66-9278-40e5be85003d)

The map shows bomb blocks, so i assume that's wrong? I have no clue which one of those are shoot blocks, I assume the top ones?

Reply:
You can shoot these blocks.